pom.xml

```diff
@@ -90,6 +91,12 @@
             <version>3.27.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Can you please explain why you're adding this dependency and how it is related with the prompt change?

Reply:
Hi @mariofusco - It is a leftover, I added it while implementing a _conversational_ test.
Will remove it now, and will add it later when I am ready with the test.